src/commands/waw/workbench/open.ts

```diff
@@ -74,7 +74,7 @@ export default class ConnectedAppCreate extends SfdxCommand {
 
     const accessToken = conn.accessToken;
     const instanceUrl = conn.instanceUrl;
-    const serverUrl = `${instanceUrl}/services/Soap/u/41.0`;
+    const serverUrl = `${instanceUrl}/services/Soap/u/46.0`;
```

Comment:
What if you did something like...

```
const version = await conn.retrieveMaxApiVersion();
const serverUrl = `${instanceUrl}/services/Soap/u/${version}`;
```

That way this doesn't have to be updated every time.

Reply:
If you don't want to do an extra api call, then at the minimum you could do `conn.getApiVersion()` which is what the current connection is using.

Reply:
Thanks @amphro, good points all around - I will update